src/client/src/components/buttons/buttons.tsx

```diff
@@ -2,7 +2,7 @@ import { forwardRef } from "react";
 import { useTranslation } from "react-i18next";
 import CloseIcon from "@mui/icons-material/Close";
 import { Button } from "@mui/material";
-import { Check, Pencil, Plus, Save, Trash2 } from "lucide-react";
+import { Check, DownloadIcon, Pencil, Plus, Save, Trash2 } from "lucide-react";
```

Comment:
REQ:  Icon aus dem [Figma](https://www.figma.com/design/cEiOoOazAQZqpRY92ZhBeO/swissgeol?node-id=7281-56716&node-type=canvas&t=UFHoWd98rI4LyBSD-0) nehmen. Pages => Tabelle.
Entweder gibt es das passende bei lucide. Ansonsten laden wir das svg dann jeweils aus dem Figma runter und erstellen ein neues Icon in src/assets/icons und binden es so ein "...   ./assets/icons/polygon.svg?react"